src/main/java/edu/kis/powp/jobs2d/panel/JPanelMouseControl.java

```diff
@@ -39,9 +39,10 @@ private static void startListener() {
         jPanel.addMouseListener(new MouseListener() {
             @Override
             public void mouseClicked(MouseEvent e) {
-                MoveFeature c1 = new MoveFeature();
+
                 System.out.println("mouseClicked, X=" + e.getX() + " ,Y=" + e.getY() + " ,Width=" + jPanel.getWidth() + " ,Height=" +
                         jPanel.getHeight() + " ,Count=" + e.getClickCount());
+               MovmentManager c1 = new MovmentManager();
```

Comment:
What is the role of c1? Seem to be only temporary.

Reply:
Changed c1 to mouseclick.